src/sentry/api/bases/organization_events.py

```diff
@@ -14,6 +14,9 @@
     'issue_title': {
         'aggregations': [['anyHeavy', 'title', 'issue_title']],
     },
+    'last_event': {
+        'aggregations': [['max', 'id', 'last_event']],
```

Comment:
I wanted to use [argMax](https://clickhouse-docs.readthedocs.io/en/latest/agg_functions/#argmax-arg-val) here to get the event_id for the max timestamp, but ran into a variety of errors when I tried that. I'd like to circle back on this once the modal is a bit more feature complete.

Reply:
This works for me - although my intention with the org event details endpoint here https://github.com/getsentry/sentry/pull/13553 is that you request`/latest` and get the entire latest event

Reply:
The latest endpoint could work too. 

I could have the modal trigger off of a `groupSlug` parameter. When present, the modal can  make a request for with the latest event endpoint instead of an event by id.